java/connector-node/risingwave-source-cdc/src/main/java/io/debezium/connector/postgresql/connection/pgoutput/PgOutputMessageDecoder.java

```diff
@@ -808,6 +808,21 @@ private Table resolveRelationFromMetadata(PgOutputRelationMetaData metadata) {
                 editor.defaultValueExpression(columnMetadata.getDefaultValueExpression());
             }
 
+            /* patch code */
+            // Check if this column is an enum type and set enum values
+            if (columnMetadata.getPostgresType().getEnumValues() != null) {
+                List<String> enumValues = columnMetadata.getPostgresType().getEnumValues();
+                if (!enumValues.isEmpty()) {
+                    editor.enumValues(enumValues);
```

Comment:
Caveat:
* The actual list of values seem currently unused. Only whether the field is set to non-null value.
* If some logic uses these values in the future, it can be inaccurate - when upstream PostgreSQL uses `ALTER TYPE` that adds a new variant to this enum, the list here won't be updated to keep in sync

Maybe we shall add this comment to the code here or the usage below